resources/github-actions.md

```diff
@@ -45,6 +45,61 @@ There is a link provided at the end of every C++ and Python build job where the
 
 ![](/assets/images/downloads.png)
 
+## Using CI Artifacts in Other PRs
+
+For changes that cross library boundaries, it may be necessary to test a pull request to one library with changes from a pull request to another library.
+Consider the overall RAPIDS dependency graph when testing.
+For example, if you are testing artifacts from an RMM PR rmm#A in cuML, you probably also need to create a cuDF PR cudf#B that uses the artifacts from rmm#A, and then your cuML test PR will need to include the artifact channels for both rmm#A and cudf#B.
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick (non-blocking):** ‏ I think it helps readability to put the PR IDs in back quotes.

```suggestion
For example, if you are testing artifacts from an RMM PR `rmm#A` in cuML, you probably also need to create a cuDF PR `cudf#B` that uses the artifacts from `rmm#A`, and then your cuML test PR will need to include the artifact channels for both `rmm#A` and `cudf#B`.
```